software/coderscreen.yml

```
@@ -0,0 +1,13 @@
+name: "CoderScreen"
+website_url: "https://www.coderscreen.com/"
+source_code_url: "https://github.com/CoderScreen/coderscreen"
+description: "Modern, open-source technical hiring platform"
```

```suggestion
description: "Modern, open-source technical hiring platform."
```
Adding a dot to the end